packages/block-editor/src/store/selectors.js

```diff
@@ -341,6 +341,17 @@ export function getBlockCount( state, rootClientId ) {
  * @property {number} offset       The selected block attribute offset.
  */
 
+/**
+ * Returns the current selection object.
+ *
+ * @param {Object} state Block editor state.
+ *
+ * @return {Object} Selection object
+ */
+export function getSelection( state ) {
```

Comment:
This selector exposes the selection state, which means we can not change its internal shape later. These expose flags like is isMultiSelecting, isEnable which have very specific internal usages. Wouldn't getSelectionStart + getSelectionEnd be an option?

Reply:
Right, might be better!